common/src/main/java/org/conscrypt/metrics/CipherSuite.java

```diff
@@ -68,6 +70,10 @@ public int getId() {
     }
 
     public static CipherSuite forName(String name) {
+        if ("SSL_RSA_WITH_3DES_EDE_CBC_SHA".equals(name)) {
```

Comment:
This seems to be unrelated to this PR.

Reply:
Sorry, I'll update the commit message!

It's part of the same overall issue which is to many things showing up as UNKNOWN.

For hysterical reasons the JSSE standard names document insists on calling that suite `SSL_RSA_WITH_3DES_EDE_CBC_SHA` but the rest of the world uses `SSL_RSA_WITH_3DES_EDE_CBC_SHA`...

Reply:
Done